deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml

```diff
@@ -40,3 +43,8 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      volumes:
+      - hostPath:
+          path: /var/run/dbus
+          type: Directory
```

Comment:
We should remove this type assertion. The pod will fail to run if there's no such directory on the host.